dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ExecutorController.java

```diff
@@ -116,13 +116,13 @@
     public Result startProcessInstance(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
                                        @ApiParam(name = "projectCode", value = "PROJECT_CODE", required = true) @PathVariable long projectCode,
                                        @RequestParam(value = "processDefinitionCode") long processDefinitionCode,
-                                       @RequestParam(value = "scheduleTime", required = false) String scheduleTime,
-                                       @RequestParam(value = "failureStrategy", required = true) FailureStrategy failureStrategy,
+                                       @RequestParam(value = "scheduleTime") String scheduleTime,
+                                       @RequestParam(value = "failureStrategy") FailureStrategy failureStrategy,
                                        @RequestParam(value = "startNodeList", required = false) String startNodeList,
                                        @RequestParam(value = "taskDependType", required = false) TaskDependType taskDependType,
                                        @RequestParam(value = "execType", required = false) CommandType execType,
-                                       @RequestParam(value = "warningType", required = true) WarningType warningType,
-                                       @RequestParam(value = "warningGroupId", required = false) int warningGroupId,
+                                       @RequestParam(value = "warningType") WarningType warningType,
+                                       @RequestParam(value = "warningGroupId") int warningGroupId,
```

Comment:
This part is not restored.

Reply:
> ```
> @RequestParam(value = "warningGroupId") int warningGroupId,
> ```
> 
> This part is not restored.

@zhuangchong If it's not required, we better set a default value, Could you offer a value?

Reply:
@zhuangchong Do you know what the default value of int, I need it to finish testStartProcessInstanceWithRequiredParams test.
Thanks

Reply:
@zhuangchong, null cannot be assigned to int. Please provide a default value.
Thanks

Reply:
@zhuangchong I used default as 0 for warningGroupId